core/pom.xml

```diff
@@ -30,6 +30,10 @@
          <artifactId>jgroups</artifactId>
       </dependency>
 
+      <dependency>
```

Comment:
Ha! Found why I decided to add it to the core....

Note that default configs are placed in the `core` (I'm not sure if this is the right place for them or not). We should support Kubernetes out of the box, so I added `default-jgroups-kubernetes.xml`. Since the configuration requires `KUBE_PING` in the classpath, we need to add this dependency to core.